agent/exploits/cve_2024_4439.py

```diff
@@ -0,0 +1,35 @@
+"""Agent Asteroid implementation for CVE-2024-4439"""
+
+import re
+
+from agent import definitions
+from agent import exploits_registry
+from agent.exploits import webexploit
+
+# Affected Versions: 6.5 – 6.5.1, 6.4 – 6.4.3, 6.3 – 6.3.3, 6.2 – 6.2.4, 6.1 – 6.1.5, 6.0 – 6.0.7
+VULNERABLE_VERSION = "6\.([0-5]|0\.[1-5]|1\.[1-5]|2\.[1-4]|3\.[1-3]|4\.[1-3]|5\.1)"
```

Comment:
Use version ranges and version comparison. This is really not readable.

Reply:
then I need to re implement the exploit and not inherit from the existing one

Reply:
maybe adabte the base exploit

Reply:
> maybe adabte the base exploit

do u have any idea about the base exploit ?